script/Arcueid/Day 7/07_00_ARC07_3G.txt

```diff
@@ -624,7 +624,7 @@ Or rather, I'd appreciate it if you'd stop looking down on one of the most law-a
 -- Page 474, Offset 17748.
 -- 「いい？　志貴は生物の死は視やすくて、鉱物の死は視にくいって言ってたでしょ？
 "Are you ready?
- Shiki, you said that you could make out the lines of death on living things
+ you said that you could make out the lines of death on living things
```

Comment:
Vote to keep Shiki since she says it out loud here. If we do delete it, need to promote You to caps

Reply:
I just took it out because arc doesn't sound as serious as how the line is, either that or maybe changing "are you ready?" would lighten it, to votes it is then.